test/dynamo/test_generator_stop.py

```diff
@@ -0,0 +1,69 @@
+# Owner(s): ["module: dynamo"]
+
```

Comment:
do you want this to be a general test_generator file instead?

Reply:
>  I see, you're mirroring the cpython tests

Exactly.